packages/replayio/src/utils/initialization/initialize.ts

```diff
@@ -42,6 +43,8 @@ export async function initialize({
     npmUpdateCheck = { hasUpdate: undefined },
   ] = await promises;
 
+  await logger.identify(accessToken);
+
```

Comment:
```suggestion
```

Reply:
Once this extra line is removed, I think this PR can be merged and I'll follow up with PRO-797

Reply:
Whoops, I left it accidentally.